src/common.ts

```diff
@@ -173,3 +173,15 @@ export abstract class Kommand extends Command {
     return eval(`var o = ${input}; o`)
   }
 }
+
+/**
+ * An iteration-order-safe version of lodash.values
+ *
+ * @param object The object containing the values
+ * @param fields The field names to pick in the right order
+ * @returns The values in the same order as the fields
+ * @see https://lodash.com/docs/4.17.15#values
+ */
+ export function pickValues(object: any, fields: string[]): any[] {
+  return fields.map(f => object[f])
```

Comment:
Are you handling this kind of path `path.to.property` somewhere else?

Reply:
Yes, it is used by the `CSVDumper` in `collection:export`. But, since this function is very generic, I wanted to make it live in some sort of `utilities.js` file. Since it doesn't exist, I've put it in `common.js`.